localisation/00_zuwc_united_localisation.csv

```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
+GetBodyQualityResponse_normal;It looks fine. You spot sharp angles and slight asymmetry, but it's barely noticeable. It is indeed a good vessel for you. You look at the carver, who gives you a smile in return.;;Er sieht gut aus. Ihr k�nnt spitze Winkel und eine leichte Asymmetrie erkennen, aber es f�llt kaum auf. Es ist in der Tat ein gutes Gef�� f�r Euch. Ihr betrachtet den Skulpteur wohlwollend, der Ihnen im Gegenzug ein L�cheln schenkt.;;;;;;;;;;;x
+spiritbinding_soul_transferred_tooltip;Your soul is successfully transferred.;;Eure Seele wurde erfolgreich �bertragen.;;;;;;;;;;;x
+spiritbinding_soul_trapped_tooltip;Your soul is trapped.;;Eure Seele ist gefangen.;;;;;;;;;;;x
+EVTOPTA_WCRAC_510;What went wrong?!;;Was ist schief gelaufen?!;;;;;;;;;;;x
+EVTDESC_WCRAC_510;The spiritbinders gather around you, muttering their mantra. You feel like you're falling. Trying to find the ground under your feet, you look down and see your body. [Root.GetSpiritbindingOutcome];;Die Geistbinder versammeln sich um Euch und murmeln ihr Mantra. Ihr f�hlt Euch, als w�rdet Ihr fallen. Ihr versucht, den Boden unter Euren F��en zu finden, schaut nach unten und seht Euren K�rper. [Root.GetSpiritbindingOutcome];;;;;;;;;;;x
+GetSpiritbindingOutcome_transferred;Your soul floats toward your new vessel. Finally, a flash. With your new eyes, you watch the old empty body falling to its knees. ;;Eure Seele flie�t auf Euer neues Gef�� zu. Endlich, ein Lichtblitz. Mit Euren neuen Augen seht Ihr zu, wie der alte leere K�rper auf die Knie f�llt. ;;;;;;;;;;;x
+GetSpiritbindingOutcome_trapped;You float farther and farther away, reaching the firmament. At the last moment of your life, you witness the giant black whirlwind among the clouds.;;Ihr schwebt immer weiter weg und erreicht schlie�lich das Himmelszelt. Im letzten Moment Eures Lebens werdet Ihr Zeuge des riesigen schwarzen Wirbelsturms zwischen den Wolken.;;;;;;;;;;;x
+build_new_vessel;Build New Vessel;;Neues Gef�� fertigen lassen;;;;;;;;;;;x
+build_new_vessel_desc;Ancient secrets are known to your people. Lifeless rock could be animated, providing a willing or unwilling soul could be captured within. Thus, you're able to transfer your soul to a new vessel, if the current one doesn't suit you anymore. You will be rejuvenated and lose all the body-related traits.;;Eurem Volk sind uralte Geheimnisse bekannt. Lebloser Fels kann belebt werden, vorausgesetzt, eine bereitwillige oder unwillige Seele kann darin gefangen werden. So w�rt Ihr in der Lage, Eure Seele in ein neues Gef�� zu �bertragen, wenn Euch das jetzige nicht mehr passt. Dadurch w�rdet Ihr verj�ngt und w�rdet alle k�rperbezogenen Eigenschaften verlieren.;;;;;;;;;;;x
```

Comment:
```suggestion
build_new_vessel_desc;Ancient secrets are known to your people. Lifeless rock could be animated, providing a willing or unwilling soul could be captured within. Thus, you're able to transfer your soul to a new vessel, if the current one doesn't suit you anymore. You will be rejuvenated and lose all body-related traits.;;Eurem Volk sind uralte Geheimnisse bekannt. Lebloser Fels kann belebt werden, vorausgesetzt, eine bereitwillige oder unwillige Seele kann darin gefangen werden. So w�rt Ihr in der Lage, Eure Seele in ein neues Gef�� zu �bertragen, wenn Euch das jetzige nicht mehr passt. Dadurch w�rdet Ihr verj�ngt und w�rdet alle k�rperbezogenen Eigenschaften verlieren.;;;;;;;;;;;x
```

Reply:
searched the whole text for your change, found it, and when i wanted to comment on it, i read your comment 😅 done

Reply:
Haha, sorry, it was only when I saw your first done that I realized it didn't highlight the bits I changed.